### PR TITLE
Document the browser specifications we test against

### PR DESCRIPTION
**Pull request by brown-a2**
Add in some guidance around what browsers we test our websites against.
I've linked to GDS so that this continues to be updated with ever GDS
review (happens twice annually).

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -29,6 +29,15 @@ This covers a lot of ground. For now I'm again, going to be referencing Google's
 We currently do not have a team style guide for using Git, however, GDS provide a useful default for us to follow:
 * [Working with Git](https://gds-way.cloudapps.digital/standards/source-code.html#working-with-git) (External site)
 
+### Testing our applications
+
+Every website, component, or app that we build with a web interface, should be tested against a suite of different browsers and devices. This is to ensure we are meeting a minimum expected standard across government applications and provide a reasonable level of accessibility for all. We currently follow GDS specifications on what browsers to test on,
+
+* [Designing for different browsers and devices](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices) (External site)
+* [Spreadsheet version of GDS list](https://docs.google.com/spreadsheets/d/1zGvXsgNIoMhynFtUV-WS0lqq5QNGY9mIWBzJ3dbIPfI/edit#gid=0) (Goog Doc - check it is up-to-date)
```
Comment (wilson1000-MoJ) [inline index.md]: Is this a typo dude: Goog Doc

**Review by wilson1000-MoJ — APPROVED** [review]
This resource you've made is one of my favourite things. Thank you Adam, some real good stuff going in 👍

**Review by wilson1000-MoJ — APPROVED** [review]

File: index.md (hunk 1 above)
Comment (brown-a2) [reply index.md]: Thanks kind of you to say! Please feel free to contribute as well.

**Comment by BNewing** [comment]
Looks good to me :) yay for putting the spreadsheet in! I'll aim to keep it up to date. 

Whilst we're adding in things to test against, it would be good to add in the [GDS Guidance on testing with Assistive Technology](https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies) too. I wonder if there's one for performance out there anywhere too...

**Review by BNewing — APPROVED** [review]
Looks great to me :) yes it would be good to discuss our team approach's approach, particularly now that we can't access the Accessibility Bars. I'll have a think and ask around! If we decide we do want to use the free AT in the meantime, we can add the link to that guidance later on post-discussion.

**Pull request merged**